pgadmin-env.list

```diff
@@ -0,0 +1,3 @@
+PGADMIN_DEFAULT_EMAIL=jamie.maynard@me.com
+PGADMIN_DEFAULT_PASSWORD=8lScP10eY1L3oeXbBM1E
```

Comment:
oh no! 😱

Reply:
Reset back to the Master version... my local environment is using a Docker version of Postgres.